main/src/ca/uwaterloo/flix/language/phase/unification/EffUnification3.scala

```diff
@@ -52,12 +52,15 @@ object EffUnification3 {
     *
     * Note: Treats `Type.Error` as a constant, i.e. only equal to itself. Hence, it is better to drop equations that contain `Type.Error`.
     */
-  def unifyAll(eqs: List[TypeConstraint.Equality], scope: Scope, renv: RigidityEnv)(implicit flix: Flix): Result[Substitution, List[TypeConstraint]] = {
+  def unifyAll(eqs0: List[TypeConstraint.Equality], scope: Scope, renv: RigidityEnv)(implicit flix: Flix): Result[Substitution, List[TypeConstraint]] = {
```

Comment:
The documentation was already outdated, but you can change `eqns` in the doc comment to `eqs0`